muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/records/body/application/vops.tsx

```diff
@@ -19,30 +21,32 @@ interface VopsState {
 }
 
 class Vops extends React.Component<VopsProps, VopsState> {
-  render(){
-    if (this.props.records.location !== "vops"){
+  render() {
+    if (this.props.records.location !== "vops") {
       return null;
-    } else if (this.props.vops.status === "ERROR"){
+    } else if (this.props.vops.status === "ERROR") {
       //TODO: put a translation here please! this happens when messages fail to load, a notification shows with the error
```

Comment:
We haven't yet fixed these TODO's, maybe we should... using the this.props.i18n.text.get in order to put something more appropiate rather than just ERROR in uppercase